nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/opvalidation/LayerOpValidation.java

```diff
@@ -1466,5 +1471,183 @@ public void testBiasAdd_nchw_nhwc() {
             String err = OpValidation.validate(tc);
             assertNull(err);
         }
+    }
+
+
+    @Test
+    public void LSTMLayerTestCase1() {
+
+        int miniBatchSize = 10;
+        int numUnits = 10;
+        int maxTSLength = 64;
+        int sL = 10; //small just for test
```

Comment:
Let's change these to make sure they are all different values - not all the same value.
The reason is simple: with multiple values being the same, we could easily miss a major bug.
Suppose one of the outputs is supposed to have shape [minibatch, numUnits] but instead it returns the wrong shape [numUnits, minibatch]. If minibatch and numUnits are the same size, it's impossible to detect this problem in your test.
This is a general principle when testing: try and keep the hyperparameters that determine the input and output shapes (minibatch, numUnits, sequence length, etc) different to help us catch shape problems.

For example, something like this:
```
        int miniBatchSize = 5;
        int numUnits = 7;
        int maxTSLength = 64;
        int sL = 10; //small just for test
```

This also applies to the other 2 tests below.